### PR TITLE
add remove/object api document

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

I have created the remove and get-object API(RPC) documents for the public document.

### Environment:

- Go Version: 1.17.5
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.13.8

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/api/remove.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+      }
+  }
+  ```
+
+  - Object.Location
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |name|string| | the name of vald agent pod where the request vector is removed. |
+    |uuid|string| | the ID of the removed vector. It is the same as an `Object.ID`. |
+    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed |
```
Comment (kmrmt) [inline docs/api/remove.md:84]: forgot **.**
```suggestion
    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed. |
```

File: docs/api/remove.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+  - Object.StreamLocation
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |location|Object.Location| | the information of Object.Location data. |
+    |status|google.rpc.Status| | the status of google RPC |
+
+  - Object.Location
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |name|string| | the name of vald agent pod where the request vector is removed |
```
Comment (kmrmt) [inline docs/api/remove.md:173]: same
```suggestion
    |name|string| | the name of vald agent pod where the request vector is removed. |
```

File: docs/api/remove.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+    |uuid|string| | the ID of the removed vector. It is the same as an `Object.ID`. |
+    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed |
```
Comment (kmrmt) [inline docs/api/remove.md:175]: same
```suggestion
    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed. |
```

File: docs/api/remove.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+  - Object.Locations
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |location|Object.Location| repeated(Array[Object.Location]) | the list of Object.Location. |
+
+  - Object.Location
+    |field|type|label|desc.|
+    |:---:|:---|:---|:---|
+    |name|string| | the name of vald agent pod where the request vector is removed |
```
Comment (kmrmt) [inline docs/api/remove.md:276]: same
```suggestion
    |name|string| | the name of vald agent pod where the request vector is removed. |
```

File: docs/api/remove.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+    |uuid|string| | the ID of the removed vector. It is the same as an Object.Vector |
```
Comment (kmrmt) [inline docs/api/remove.md:277]: same
```suggestion
    |uuid|string| | the ID of the removed vector. It is the same as an Object.Vector. |
```

File: docs/api/remove.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed |
```
Comment (kmrmt) [inline docs/api/remove.md:278]: same
```suggestion
    |ips|string| repeated(Array[string]) | the IP list of `vald-agent` pods where the request vector is removed. |
```

**Review by kmrmt — CHANGES_REQUESTED** [review]

**Review by kevindiu — DISMISSED** [review]

**Review by kmrmt — APPROVED** [review]

**Review by kmrmt — APPROVED** [review]

**Review by kevindiu — APPROVED** [review]

**Pull request merged**